client/jetpack-connect/style.scss

```diff
@@ -656,3 +656,25 @@
 .jetpack-connect__navigation {
 	text-align: center;
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 970px;
+}
+
+.jetpack-connect__plan-info-illustration {
+	margin: 30px 0 10px;
+	max-width: 25%;
+}
+
+.jetpack-connect__plan-info-buttons {
+	display: block;
+	margin-bottom: 20px;
+	text-align: center;
+	width: 970px;
+
+	.button {
+		margin-right: 10px;
```

Comment:
Having this for both buttons will make the buttons not precisely aligned to the center. How about adding `margin-left` to every `.button` but the first one (for example by using the `.button + .button` selector)?